layouts/partials/search-script.html

```diff
@@ -4,27 +4,122 @@
    */}}
 {{ $currentSection := .FirstSection.Params.versionId | default .Site.Params.versions.current }}
 {{ if  .Site.Params.docSearch }}
-  <script src="https://cdn.jsdelivr.net/npm/@docsearch/js@3.2.1"></script>
-  <script type="text/javascript">
-    docsearch({
-      appId: '{{ .Site.Params.docSearch.appId }}',
-      apiKey: '{{ .Site.Params.docSearch.apiKey }}',
-      indexName: '{{ .Site.Params.docSearch.indexName }}',
-      placeholder: 'Search DevDocs',
-      searchParameters: {
-        facetFilters: [
-          'version:{{ $currentSection }}'
-        ],
-      },
-      container: '#docsearch-container',
-    });
-  </script>
+    <script src="https://cdn.jsdelivr.net/npm/@docsearch/js@3.2.1"></script>
+    <script type="text/javascript">
+
+        const hookReferential = [
+            {{ with .Site.GetPage "/8/modules/concepts/hooks/list-of-hooks" }}
```

Comment:
Could we have it configurable? It should use the current version of the docs (what about v9 in the future?), and we should probably have a chance to change this URL from version to version if needed.